decksite/data/archetype.py

```diff
@@ -17,7 +17,7 @@ class Archetype(Container, NodeMixin):
 
 BASE_ARCHETYPES: Dict[Archetype, Archetype] = {}
 
-def load_archetype(archetype: Union[int, str], season_id: int = None) -> Archetype:
+def load_archetype(archetype: Union[int, str], season_id: int = None, tournament_only: bool = False) -> Archetype:
```

Comment:
What about always pulling out both sets of stats? If they're going to be precalculated anyway it doesn't really cost anything in terms of user time and we can skip this parameter.

Reply:
I've pushed a change to do this. The result is the tournament filtering getting pushed down the pipeline to later in the view processing. The effect of making the file data/archetype.py agnostic to this is contained in [this diff](https://github.com/PennyDreadfulMTG/Penny-Dreadful-Tools/pull/5907/commits/451c745ffc36992d1b634263f097b7b18b40d831).

Reply:
That sounds good if it's not any slower (which I theorize it should not be). I'm going to merge this as soon as I'm done with migrating to scryfall from mtgjson3. Should hopefully not be too long. I'm down to the level of dealing with weird card types like "Scheme" now. Most everything normal is working.

Reply:
The current version passes both sets of info to the template and lets the template decide which to display. The code is a little uglier now, but it's more flexible in theory, for example if a page wants to have both tournament and all results side by side, or if a future version of this will load both and switch via js instead.